libs/utils/utils.js

```diff
@@ -614,6 +614,12 @@ export function decorateLinks(el) {
     appendHtmlToLink(a);
     a.href = localizeLink(a.href);
     decorateSVG(a);
+    if (window.location.hostname.includes('.stage.adobe.com')
```

Comment:
One of the checks seems redundant. This should work as well:
```
if (window.location.hostname.includes('.stage.adobe.com') && !a.hostname.includes('.stage.adobe.com')) {
  a.href = a.href.replace('.adobe.com', '.stage.adobe.com');
}
```

Reply:
That is true 👍 Before I was thinking that we have links pointing to stage already, but they won't pass the `!a.hostname.includes('.stage.adobe.com'))` check anyway